Formula/clang-format.rb

```diff
@@ -83,6 +86,6 @@ def install
         shell_output("#{bin}/clang-format -style=Google test.c")
 
     ENV.prepend_path "PATH", bin
-    assert_match "test.c", shell_output("git clang-format")
+    assert_match "test.c", shell_output("git clang-format", 1)
```

Comment:
This doesn't seem correct. Why is it returning a non-zero code?

Reply:
Newer versions of `git-clang-format` do this to signal that some files were not formatted correctly (which is the case in this test). The corresponding [patch](https://github.com/llvm/llvm-project/commit/edbb8a843c130e60d71cb73e56a33d5ba2cc0ec9) landed in LLVM 14, but [due to a bug](https://github.com/llvm/llvm-project/issues/54758) which was only fixed in the 15.x release series, LLVM 15 is the first version to actually incorporate these return codes.

Reply:
Can we fix the file formatting instead?

Reply:
The file is formatted incorrectly on purpose (see also: line 79), as (as far as I can see) the point of the test is that (git-)clang-format recognizes and fixes the incorrect formatting.

Reply:
Ah right, I missed the `git` prefix. Looks good then